src/main/java/pl/gajoch/layview/graphics3d/VectorProperties.java

```diff
@@ -0,0 +1,30 @@
+package pl.gajoch.layview.graphics3d;
+
+import javafx.scene.paint.Color;
+
+public class VectorProperties {
+    public double tipLen, tipRadius, radius, lenScale;
+    public Color SpecularColor;
+    public int divisions;
+
+    public VectorProperties(double tipLen, double tipRadius, double radius, double lenScale, Color SpecularColor, int divisions) {
+        this.tipLen = tipLen;
+        this.tipRadius = tipRadius;
+        this.radius = radius;
+        this.SpecularColor = SpecularColor;
+        this.lenScale = lenScale;
+        this.divisions = divisions;
+    }
+
+    public VectorProperties(double tipLen, double tipRadius, double radius, double lenScale, Color SpecularColor) {
+        this(tipLen, tipRadius, radius, lenScale, SpecularColor, 64);
+    }
+
+    public VectorProperties(double tipLen, double tipRadius, double radius, double lenScale) {
+        this(tipLen, tipRadius, radius, lenScale, Color.BLACK);
+    }
+
+    public VectorProperties() {
```

Comment:
Example objects should be created in test cases. Parameter-less constructor is not place for such things.